modules/benchmark/R/load_data.R

```diff
@@ -34,13 +34,27 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
     fcn <- match.fun(fcn1)
   } else if (exists(fcn2)) {
     fcn <- match.fun(fcn2)
+  } else if (!exists(fcn1) & !exists(fcn2)) { 
```

Comment:
Maybe add a `& require(bd)` here. This will return FALSE if the library is not found and it will skip this section. This allows us to make the browndog library be a suggestion and it will work wether or not somebody has it installed.